packages/components/src/components/hds/code-editor/index.hbs

```diff
@@ -0,0 +1,49 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div
+  class={{this.classNames}}
+  {{! @glint-expect-error - https://github.com/josemarluedke/ember-focus-trap/issues/86 }}
+  {{focus-trap isActive=this._isFullScreen}}
+  {{this._handleEscape}}
+  ...attributes
+>
+  {{#if (and this._isSetupComplete (or this.hasActions (has-block)))}}
+    <div class="hds-code-editor__header">
+      <div class="hds-code-editor__header-content">
+        {{yield (hash Title=(component "hds/code-editor/title") Description=(component "hds/code-editor/description"))}}
+      </div>
+
+      {{#if this.hasActions}}
+        <div class="hds-code-editor__header-actions">
+          {{#if @hasCopyButton}}
+            <Hds::Copy::Button
+              class="hds-code-editor__button hds-code-editor__copy-button"
+              @isIconOnly={{true}}
+              @size="small"
+              @text="Copy"
+              @textToCopy={{this._value}}
+            />
+          {{/if}}
+          {{#if @hasFullScreenButton}}
+            <Hds::CodeEditor::FullScreenButton
+              @isFullScreen={{this._isFullScreen}}
+              @onToggleFullScreen={{this.toggleFullScreen}}
+            />
+          {{/if}}
+        </div>
+      {{/if}}
+    </div>
+  {{/if}}
+  <div
+    class="hds-code-editor__editor"
+    {{hds-code-editor value=@value language=@language onInput=this.onInput onSetup=this.onSetup}}
+  />
+  {{#unless this._isSetupComplete}}
+    <div class="hds-code-editor__loader" aria-live="polite" role="status">
+      <Hds::Icon @name="loading" @size="24" />
```

Comment:
Issue: sorry, I think this got lost in the last comment, but you also need to add a visually hidden span inside the div that says "loading" otherwise there is no text in the aria-live region.

```html
<span class="sr-only">Loading</span>
```